test/codeu/chat/TestRunner.java

```diff
@@ -30,14 +30,14 @@ public static void main(String[] args) {
              codeu.chat.util.UuidTest.class,
              codeu.chat.util.store.StoreTest.class
          );
-      for (final Failure failure : result.getFailures()) {
-         System.out.println(failure.toString());
-      }
 
-      if (!result.wasSuccessful()) {
+      if (result.wasSuccessful()) {
+        System.out.println(result.wasSuccessful());
```

Comment:
As just printing "TRUE" is not that meaningful to a user (it has bothered me for a long time) can we change it to be a more meaningful message?